src/components/ArticlePage/MarkdownHtml.js

```diff
@@ -57,8 +92,12 @@ const Html = styled.div`
   }
   h3 {
     ${H5.styles}
-    margin: ${baselineSpacing}px 0;
+    margin: ${5 * baselineSmall}px 0 ${baselineSmall}px;
```

Comment:
I really dislike the three-value shorthand, could you explicitly specify all the four dimensions?